libs/sdk-ui-tests-e2e/cypress/integration/01-sdk-ui-dashboard/insightOnDashboard.spec.ts

```diff
@@ -79,3 +87,95 @@ describe("Insights on dashboard", () => {
         },
     );
 });
+
+describe("Date filtering on insight", () => {
+    beforeEach(() => {
+        Navigation.visit("dashboard/dashboard-date-filtering-on-insight-scenario");
+        editMode.isInEditMode(false).edit().isInEditMode();
+    });
+
+    it(
+        "remember last setting after selecting another insight",
+        { tags: ["checklist_integrated_tiger"] },
+        () => {
+            widget.waitChartLoaded();
+
+            widgetConfiguration.open().openConfiguration().selectDateDataset(DATASET_CREATED);
+
+            secondWidget.hasTitle("Column with two measures by date");
+            secondWidgetConfiguration.open().openConfiguration().isDateFilterExist();
+
+            widgetConfiguration.hasDatasetSelected(DATASET_CREATED);
+        },
+    );
+
+    it("disable date filter", { tags: ["checklist_integrated_tiger"] }, () => {
+        widget.waitChartLoaded();
+        widgetConfiguration.open().openConfiguration()
+            .isDateFilterExist(true).disableDateFilter().isDateFilterExist(false);
+    });
+
+    it("enable date filter", { tags: ["checklist_integrated_tiger"] }, () => {
+        widget.waitChartLoaded();
+        widgetConfiguration.open().openConfiguration().
+        disableDateFilter().isDateFilterExist(false).close();
+
+        widgetConfiguration.open().openConfiguration().
+        enableDateFilter().isDateFilterExist(true);
+    });
+
+    it("change filter on added insight", { tags: ["checklist_integrated_tiger"] }, () => {
+        widget.waitChartLoaded();
+        widgetConfiguration.open().openConfiguration()
+            .selectDateDataset(DATASET_CREATED);
+
+        widget
+            .waitChartLoaded()
+            .getChart()
+            .getDataLabelValues()
+            .should("deep.equal", ["$4,108,360.80", "$2,267,528.48", "$3,461,373.87"]);
+    });
+
+    it("remember disabled date filter after saving", { tags: ["checklist_integrated_tiger"] }, () => {
+        secondWidget.waitChartLoaded();
+        secondWidgetConfiguration.open().openConfiguration()
+            .disableDateFilter().isDateFilterExist(false);
+
+        dateFilter.openAndSelectDateFilterByName(DateFilterValue.THIS_MONTH).apply();
```

Comment:
why do you change date filter here?

Reply:
add more action to editmode can save

Reply:
in bear, we don't need to add any action to enable save button
toggle date filter checkbox is an action, and save button should be enable if we toggle it
confirm again, is it a platform issue?